### ISSUE TITLE
[BUG] Unable to install from curseforge app

### ISSUE THREAD

**Issue by wacopiper** [Marsgames/GearHelper#138]
Trying to install 3.3.0 beta, beta 2 and 3.3.0. Unable to install from Curse Forge app. I get the following message: Unable to install dependency. A latest file could not be found for the following addon ID: 20113.

**To Reproduce**
If this bug is due to *itemLink*, *hovering*, *quest reward*, or any other way where an item can be the cause, please provide us
- Your class
- Your level
- Your spec
- What item / quest is causing this issue

**Additional context**

**Comment by MarshallBuhl** [comment]
An update for GH came out a couple days ago along with updates for several of my other addons.  Curseforge could not install the GH update, but had no problem with the others.  I tried restarting CurseForge, but that didn't help.  I removed GH from my library and tried to install a fresh copy.  No joy.  Rebooting my computer didn't help.

Occasional attempts result in a message something like "General installation error," but that isn't a very helpful message.
